### ISSUE TITLE
Question: bundel a node package

### ISSUE THREAD

**Issue by tjoskar** [egoist/tsup#635]
Hi,

I'm thinking of bundling a node server into a single file to make it "portable".

However, when reading the [documentation](https://tsup.egoist.sh/#excluding-all-packages) I see:

> If you are using tsup to build for Node.js applications/APIs, usually bundling dependencies is not needed, and it can even break things, for instance, while outputting to [ESM](https://nodejs.org/api/esm.html).

If I'm not outputting ESM, are there any other cases that could break the code? For example, how is tsup handling imports of wasm files?

Thanks!

**Comment by weyert** [comment]
Hey @tjoskar Did you ever found an answer to your question?

**Comment by tjoskar** [comment]
@weyert No, not really. I'm also wondering why ESM causes more trouble than common js when bundling. Do you have any insights?
